config/v1/types_cluster_version.go

```diff
@@ -215,6 +222,71 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionCapability enumerates optional, core cluster components.
+type ClusterVersionCapability string
+
+const (
+	// ClusterVersionCapabilityOpenShiftSamples manages the sample
+	// image streams and templates stored in the openshift
+	// namespace, and any registry credentials, stored as a secret,
+	// needed for the image streams to import the images they
+	// reference.
+	ClusterVersionCapabilityOpenShiftSamples ClusterVersionCapability = "openshift-samples"
+)
+
+// ClusterVersionCapabilitySet defines sets of cluster version capabilities.
+type ClusterVersionCapabilitySet string
+
+const (
+	// ClusterVersionCapabilitySetNone is an empty set containing no capabilities.
+	ClusterVersionCapabilitySetNone ClusterVersionCapabilitySet = "None"
+
+	// ClusterVersionCapabilitySet4_11 is the recommended set of
+	// optional capabilities for the 4.11 version of OpenShift.
+	// This list will remain the same no matter which version of
+	// OpenShift is installed.
+	ClusterVersionCapabilitySet4_11 ClusterVersionCapabilitySet = "v4.11"
+)
+
+// ClusterVersionCapabilitySets defines sets of cluster version capabilities.
+var ClusterVersionCapabilitySets = map[ClusterVersionCapabilitySet]ClusterVersionCapability{
+	ClusterVersionCapabilitySetNone: {},
+	ClusterVersionCapabilitySet4_11: {
+		ClusterVersionCapabilityOpenShiftSamples,
+	},
+}
+
+// ClusterVersionCapabilitiesSpec selects the managed set of
+// optional, core cluster components.
+// +k8s:deepcopy-gen=true
+type ClusterVersionCapabilitiesSpec struct {
+	// baselineCapabilitySet selects an initial set of
+	// capabilities, which can be extended via
+	// additionalEnabledCapabilities.
+	BaselineCapabilitySet ClusterVersionCapabilitySet `json:"baselineCapabilitySet,omitempty"`
+
+	// additionalEnabledCapabilities add additional capabilies to
```

Comment:
adds additional capabilities, not included in the baselineCapabilitySet to this cluster.

Reply:
do we care if there are entries here that happen to also show up in the baseline set?  I don't have a problem with redundant entries like that.  I've gone with:

> additionalEnabledCapabilities extends the set of managed capabilities beyond the baseline defined in baselineCapabilitySet.

to get in your baselineCapabilitySet reference, but with `extends` to get at "set addition" instead of mentioning "not included".  Does that work for you?

Reply:
Redundant entries are allowed.

Reply:
ok, marking this resolved then with my updated wording.